---
layout: archive
title: "People"
redirect_from: 
  - /people/
  - /people.html
sidebar:
  nav: "main"
---

# Project Lead/Coordinating Committee

List yourself

## Ben Tully

<img align="left" src="/images/benjtully.jpg" width="200px" style="padding-right: 15px">

**Email:** tully.bj@gmail.com

**Twitter:** [@phantomBugs](https://twitter.com/phantomBugs)

**[Website](https://bjtully.github.io/)**

**[Google Scholar](https://scholar.google.com/citations?user=UIACF0wAAAAJ&hl=en)**

**Bio:** I am a microbiologist/bioinformatian working in the marine environment. My interests involve studying microbial communities and functions through the tool of metagenomics. As part of this research, I work to develop new bioinformatic tools that capitalizes on the growing number of environmental microbial genomes.

## Mike Lee

<img align="left" src="/images/Mike-Lee-Atlantis.jpg" width="200px" style="padding-right: 15px">

**Email:** MikeLee@bmsis.org  

**Twitter: [@AstrobioMike](https://twitter.com/AstrobioMike){:target="_blank"}**  

**[Website](https://astrobiomike.github.io/research/){:target="_blank"}**  

**[Google Scholar](https://scholar.google.com/citations?user=-ONw6lsAAAAJ){:target="_blank"}**  
<br>
**Bio:** Iâ€™m a microbial ecologist/bioinformatician at NASAâ€™s Ames Research Center in northern California, USA. I get to work in all kinds of different systems ranging from the bottoms of our oceans up to the International Space Station ðŸ‘½. Having [gone through it myself](https://astrobiomike.github.io/about/){:target="_blank"}, I very much enjoy trying to help folks get through the initial steep learning curve that lots of us face when we start getting into bioinformatics. In addition to regularly helping to organize and teach at bioinformatics workshops such as [STAMPS](https://mblstamps.github.io/){:target="_blank"} and [ANGUS](https://angus.readthedocs.io/en/2019/){:target="_blank"}, I develop and maintain [Happy Belly Bioinformatics](https://astrobiomike.github.io/){:target="_blank"}, a site that aims to be a useful resource for biologists who are new to bioinformatics ðŸ™‚  

# Active Instructors

## Jake L. Weissman

<img align="left" src="/images/jlw2.jpg" style="padding-right: 15px">

**Email:** jakeweis@usc.edu

**Twitter:** @jlw_ecoevo

**[Website](https://jlw-ecoevo.github.io)** 

**[Google Scholar](https://scholar.google.com/citations?user=IaAUSiQAAAAJ&hl=en)**

**Bio:** The battle between microbes and their viruses is ancient and ongoing. Over billions of years of coevolution, microbes have evolved a great diversity of strategies to protect themselves against viral infections. What drives selection favoring a particular antiviral defense strategy? What new defense systems are still waiting to be discovered? How do defense systems coevolve with pathogens over time, and how does this coevolution feed back into the dynamics of host-virus interactions? The central theme of my previous and ongoing work is the application of tools from the fields of machine learning and complex systems science, alongside more traditional approaches from population genetics and theoretical ecology, to understand the ecology and evolution of antiviral defense strategies.


## Joy Buongiorno

<img align="left" src="/images/joyb.jpeg" style="padding-right: 15px">

**Email:** jbuongior21@gmail.com

**Twitter:** @DrJoyBuongiorno

**[Google Scholar](https://scholar.google.com/citations?user=3pa76GEAAAAJ&hl=en&oi=ao)**

**Bio:**  I am a geomicrobiologist combining a background in geoscience with a diverse toolkit of novel molecular methods, sequencing analysis, and bioinformatic tools to delve into the interactions between the biosphere and geosphere. I am currently a postdoctoral fellow at the Earth and Planets Laboratory at the Carnegie Institution for Science working on the NASA Astrobiology Institute ENIGMA Project. I apply methods at the frontier of integrative earth, life, and data science to answer big questions related to the co-evolution of the biosphere and geosphere through deep time.


## Ashley B. Cohen, M.S.

<img align="left" src="/images/CohenPhoto.jpg" width="150px" style="padding-right: 15px">

**Email:** ashleybrookcohen@gmail.com

**Twitter:** @ashbc1988

**Bio:**  I am a PhD candidate in biogeochemistry at the School of Marine and Atmospheric Sciences, Stony Brook University, NY in Dr. Gordon Taylorâ€™s Marine Microbial Ecology laboratory. I specialize in the geobiology, or the interaction of geochemical processes and microbes, of oxygen-free environments. My dissertation work at Fayetteville Green Lake, NY, is focused on how the succession of microbial carbon fixation pathways and sulfur metabolisms along redox gradients influences stable isotope signatures. I am especially interested in how they are compartmentalized into suspended and sinking particulate matter in the water column, and ultimately, what is incorporated into sediment. I hope this will provide insight into which early earth anoxic water column processes are reflected in the geologic record. I combine microbiology techniques including Illumina sequencing, stable isotope probing, radiotracer rate measurements, qPCR, and fluorescence in situ hybridization with geochemical techniques to answer these questions.


## Harriet Alexander

<img align="left" src="/images/harriet-alexander.jpg" width="200px" style="padding-right: 15px">

**Email:** halexander@whoi.edu

**Twitter:** @nekton4plankton

**[Website](https://alexanderlabwhoi.github.io/)**

**[Google Scholar](https://scholar.google.com/citations?user=FdYCw1gAAAAJ&hl=en)**

**Bio:** I am an Assistant Scientist in the Biology Department at the Woods Hole Oceanographic Institution interested in bioinformatics and microbes in the ocean. My work focuses on characterizing the the interplay of physiological ecology and diversity in the ocean with a focus on eukaryotic plankton. Much of my research has leveraged meta-omic data to derive insights into the diversity and function of these ecosystems.



## Alexis Marshall

<img align="left" src="/images/AMarshallPhoto.jpg" width="200px" style="padding-right: 15px">

**Email:** alexis.marshall@waikato.ac.nz

**Twitter:** @lexyjayne

**Bio:** My research investigates the functional role of microbial communities in the environment. I am currently based at the University of Waikato, New Zealand as a research fellow with Prof. Craig Cary where we study the response of microbial communities within coastal sediments to ocean acidification.  I am also involved in the characterisation of microbial functional roles within the Dry Valley, Ross Ice Shelf and Ross Sea ecosystems of Antarctica. 



## Sarah K. Hu

<img align="left" src="/images/profile-slack-shu.jpg" width="200px" style="padding-right: 15px">

**Twitter:** @shu251

**[Website](https://shu251.github.io)**

**[Google scholar](https://scholar.google.com/citations?user=9CI91CMAAAAJ&hl=en)**

I am a Postdoctoral Fellow at Woods Hole Oceanographic Institution (Marine Chemistry & Geochemistry) working to reveal the ecological roles single-celled microbial eukaryotes play in marine ecosystems. My current work is set at deep-sea hydrothermal vents, where I am characterizing in situ protistan community composition and food web interactions. The majority of my work is reliant on using genetic methods to address which microbial eukaryotes are present and what are they doing in the environment.


## Ella Sieradzki

<img align="left" src="/images/Ella_Joline_out_small.jpg" width="200px" style="padding-right: 15px">

**Email:** ellasiera@berkeley.edu  

**Twitter: [@SieraVirus](https://twitter.com/SieraVirus){:target="_blank"}**  

**[Google Scholar](https://scholar.google.com/citations?user=lt32teUAAAAJ&hl=en){:target="_blank"}**  
<br>
**Bio:** I love viruses! Except for Coronaviruses. I don't love those. I'm a jack of all trades working on bacteria and viruses in the ocean and in soil. I live and breathe metagenomics, metatranscriptomics and stable isotope probing, but dabbled in amplicon work in the past. Most of all I like to know everything about interactions between viruses and their hosts, and some day it will be my main field. For now I'm a postdoc in Mary Firestone's lab. Show me your cats and dogs and we will be the best of friends.


## Liz Suter

<img align="left" src="/images/Liz_photo.jpg" width="200px" style="padding-right: 15px">

**Email:** esuter@molloy.edu  

**Twitter: [@LizzSuter](https://twitter.com/LizzSuter){:target="_blank"}**  

**[Google Scholar](https://scholar.google.com/citations?user=OhhGHjsAAAAJ&hl=en){:target="_blank"}**  

**[Website](https://works.bepress.com/elizabeth-suter/)**


<br>
**Bio:** I am an Assistant Professor at Molloy College where I teach in the Environmental Sciences. For my PhD, I worked to decipher the biogeochemical roles of microbes in the Cariaco Basin, a permanently anoxic ocean basin in the Caribbean Sea. Since becoming a professor, I have developed a passion for teaching bioinformatics to undergraduate students. I like to mix computational, bench work, and microscopy and to involve students in all aspects of my research!


## R. Eric Collins

<img align="left" src="/images/photo_rec.jpg" width="200px" style="padding-right: 15px">

**Twitter:** @rec3141

**[Website](https://cryomics.org)**

**[Google scholar](https://scholar.google.com/citations?user=St9qTsMAAAAJ&hl=en)**

<br>
**Bio:** I am an Assistant Professor and Canada Research Chair in Arctic Marine Microbial Ecosystem Services at the University of Manitoba in Winnipeg. I work on understanding the diversity, distribution, and function of Arctic marine microbial communities, especially those in frozen environments like sea ice. I am particularly interested in evolutionary questions related to the role of horizontal gene transfer in the adaptation of microbes to extreme environments.




## Arkadiy Garber

<img align="left" src="/images/IMG_4433 copy.JPG" width="200px" style="padding-right: 15px">

**Twitter:** @bioinfoark

**[Website](https://github.com/Arkadiy-Garber?tab=repositories)**

**[Google scholar](https://scholar.google.com/citations?hl=en&user=SGPloYgAAAAJ)**

<br>
**Bio:** I am bioinformatician at the University of Montana. My research interests span from geomicrobiology to bacterial endosymbiosis. My current research involves using a variety of phylogenetic approaches, as well as functional annotation, comparative genomics, and transcriptomics to better understand genomic and cellular adaptations of bacteria as they transition from a free-living to an intracellular lifestyle. I am also very interested in functional annotation more broadly, and currently developing a bioinformatics platform to allow researchers to carry out a more-targetted approach to functional annotation, using smaller and more-curated datasets that address specific questions or pathways (e.g. biofilm formation, motility, iron cycling, magnetosome formation, etc.).



## Chris Trivedi

<img align="left" src="/images/ChrisTrivedi.jpg" width="200px" style="padding-right: 15px">

**Twitter: [@Chris_Trivedi](https://twitter.com/Chris_Trivedi){:target="_blank"}**

**[Website](https://christrivedi.com){:target="_blank"}**

**[Google scholar](https://scholar.google.com/citations?user=YZtzseYAAAAJ&hl=en){:target="_blank"}**

<br>
**Bio:** I am currently working as a postdoctoral scholar in the Interface Geochemistry group of Liane G. Benning at the German Research Centre for Geosciences GFZ in Potsdam, Germany. Part of my research in this group is to understand the role that biology plays on glacier melt dynamics. While we study all manner of microorganisms that live on glaciers, we are particularly interested in snow and ice algae and their impact on changes to glacial albedo and carbon transformation processes. We use a variety of techniques to answer our research questions including targeted amplicon sequencing, metagenomic/metatranscriptomic sequencing, and metabolomics. Our current focus is on glacial ecosystems in Iceland and Greenland.




## Philip Leftwich

<img align="left" src="/images/PhilipLeftwich.jpg" width="200px" style="padding-right: 15px">

**Twitter:** [@PhilipLeftwich](https://twitter.com/PhilipLeftwich)

**[Website](https://people.uea.ac.uk/p_leftwich)**

**[Google scholar](https://scholar.google.com/citations?user=5-qU7lkAAAAJ&hl=en)**

**Bio:** I'm a lecturer at the University of East Anglia in Norwich, UK. I teach genetics, molecular biology, microbiology, and data science. My research interests are mainly focused on genetics, microbes and pest control! I use genetic engineering, CRISPR, and gene drives to developm new technologies for the control of agricultural & disease spreading insect pests. I also work on the characterisation of microbiomes in multiple species of fruit flies, where I use a variety of techniques to find out about the functional and evolutionary relationships between microbes and hosts. 


## Luis E. Valentin-Alvarado

<img align="left" src="/images/14_luisvalentin_2.jpg" width="200px" style="padding-right: 15px">

**Email: luisva@berkeley.edu**

**Twitter: @Luis_valentin6**

**[Google scholar](https://scholar.google.com/citations?hl=en&user=y7A6fQEAAAAJI)** 

**Bio:** I am a PhD student in the Jill Banfield and Dave Savage groups at UC Berkeley. The current toolkit for human genome editing could be dramatically expanded through the exploration of new CRISPR-Cas and associated proteins that have yet to be characterized, and through the development of enzymatic tools to enable controlled repair of DNA breaks. My PhD thesis aims to advance the science and technology of genome editing by discovering and harnessing new CRISPR-Cas and related systems through a combination of bioinformatics and biochemistry.

## Arianna Krinos


<img align="left" src="/images/ariannakrinos.jpg" width="200px" style="padding-right: 15px">

**Email:** akrinos@mit.edu

**Twitter:** @ariannakrinos

**[Website](https://akrinos.github.io/)**

**[Google Scholar](https://scholar.google.com/citations?user=dguBKzUAAAAJ&hl=en&oi=ao)**

**Bio:** I am a PhD student in Biological Oceanography in the MIT-WHOI Joint Program. I am co-advised by Harriet Alexander and Mick Follows, and use bioinformatics and models to study eukaryotic phytoplankton, with a penchant for coccolithophores and diatoms. I graduated from Virginia Tech with undergraduate degrees in Computer Science, Biological Sciences, and Computational Modeling and Data Analytics with a minor in math, and in the past I have done research in mathematical modeling of environmental time series data, ecological modeling of aquatic ecosystems, and computational approaches to amphibian disease ecology. I am helping with the BVCN mainly through the Reproducibility Challenge and associated tutorials for reproducing bioinformatics workflows. Happy to help with transcriptomics or general software/coding problems.

## Adelaide Rhodes

<img align="left" src="/images/adelaide_Ski.jpg" width="200px" style="padding-right: 15px">

**Email:** adelaide.rhodes@gmail.com

**Twitter:** @RareSeas

**[Website](https://www.linkedin.com/in/adelaide-rhodes-ph-d-376b9133/)** 

**[Google Scholar](https://scholar.google.com/citations?user=CNTMRigAAAAJ&hl=en)**

**Bio:** I have a big love of nature and animals.  I started out as a zoologist studying marine copepods and have transitioned into a bioinformatic zoologist working on a wide variety of human health, vector-borne disease and crop science projects.  I have a diversity of clients and interests, my two most recent papers were on innate immunity genes in the stinkbug *Halyomorpha halys* and the long non-coding RNA's in the American beaver *Castor canadensis*.  I am happy to talk to students about their career plans, feel free to contact me.


## Jacob Cram

<img align="left" src="/images/jcram.jpg" width="200px" style="padding-right: 15px">

**Email:** jcram@umces.edu

**Twitter:** @JacobACram

**[Website](https://www.umces.edu/hpl/cram-lab)** 

**[Google Scholar](https://scholar.google.com/citations?user=oytbDhYAAAAJ&hl=en&oi=ao)**

**Bio:** I am a microbial ecologist whose research focuses on entire microbial communities. I am interested both in which organisms are present and in how those those organisms affect and in turn are shaped by their environment. My group observes micro-organisms in their native environments through molecular biology techniques and microscopy. We also use statistical and mechanistic models to integrate data and better understand microorganisms in their environments, as well as perform experiments to see how entire communities respond to perturbations.

## Maria Pachiadaki 

<img align="left" src="/images/mpachiadaki.jpg" width="200px" style="padding-right: 15px">

**Email:** mpachiadaki@whoi.edu

**Twitter:** @microbiaki

**[Google Scholar](https://scholar.google.com/citations?user=cM5nYT4AAAAJ&hl=en&authuser=1)**

**Bio:** I want to unveil the mysteries of the microbial life of the aphotic ocean realm. I am particularly interested in the identity, function and activity of the dark ocean microorganisms - from the community to the single cell level - and their effects on global ecosystem processes. I also work on the development of technology for in situ studies in marine microbial ecology.

---


# Inactive Instructors

## Daan Speth

<img align="left" src="/images/DRS_photo.jpg" width="200px" style="padding-right: 15px">

**Email:** dspeth@caltech.edu

**Twitter:** @daanspeth

**[Google Scholar](https://scholar.google.com/citations?user=1xoXn04AAAAJ&hl=en)**

**Bio:** I'm microbial ecologist/bioinformatician/geobiologist interested in the way the uncultured microbial majority makes a living, and influences life and the planet around it. I apply a mix of bioinformatics, field science, and labwork to try and improve our understanding of microbial communities. I'm currently a postdoc in the lab of Prof. Victoria Orphan at Caltech, where I work on the microbes living at a hydrothermal vent system in the Gulf of California, and those in Mono Lake. In addition, I try to think of ways to leverage the ever-increasing data provided by environmental sequencing to further increase our knowledge of the microbial world.


## Elizabeth McDaniel
<img src="https://avatars1.githubusercontent.com/u/18299362?s=460&u=2759d5e6ec166f8f71fb0b1521324fe813d049a2&v=4" align="left" width="200" style="padding-right: 15px">

**Email:** elizabethmcd93@gmail.com

**Twitter:** @lizilla93

**[Website](elizabethmcd93.github.io)**

**Bio:** I am a PhD student in the Microbiology Doctoral Training Program in Dr. Katherine McMahon's lab at the University of Wisconsin - Madison. I use a combination of enrichment culture techniques and integrated genome-resolved metagenomics to investigate bacterial population dynamics of engineered wastewater systems. I am passionate about helping people solve their analytical challenges as it relates to their research. I am a certified Carpentries instructor and a co-chair of the Computational Biology, Ecology, and Evolution [(ComBEE)](https://combee-uw-madison.github.io/studyGroup/) study group on the UW-Madison campus.

## Morgan V. Evans
<img src="https://avatars2.githubusercontent.com/u/51716739?s=460&u=952436b987be5903627bc69d88a77fdd36ec1efa&v=4" align="left" width="200" style="padding-right: 15px">

**Email:** evans.2408@osu.edu

**Twitter:** @morgvevans

**[Website](https://github.com/morgvevans)**

**[Google Scholar](https://scholar.google.com/citations?user=vVUCZw0AAAAJ&hl=en&authuser=1)**

**Bio:** I am postdoctoral researcher at The Ohio State University in the Colleges of Public Health & Veterinary Preventative Medicine. My current research uses -omic tools to investigate strain level overlap between humans, animals, and the environment, as well as to reconstruct chemical/xenobiotic microbial pathways in the human and animal gut as it relates to health & disease. My PhD work focused on microbial xenobiotic metabolisms in hydraulically fractured oil and gas systems. I am eager to help people who don't have coding or bioinformatics backgrounds learn their way around the command line for their work, particularly for metagenomics and 16S rRNA sequencing analysis.
